### PR TITLE
add empty value to exceeding limmit message

### PR DESCRIPTION
**Pull request by ubbabeck**
Ref https://github.com/breez/breezmobile/pull/1090

Regarding you feedback,I was not able to pass a null param so I ended up with generating a new variable without a param.

### CODE REVIEW

File: lib/generated/breez_translations.dart [hunk 1]
```diff
@@ -95,8 +95,8 @@ abstract class BreezTranslations {
 
   /// A list of this localizations delegate's supported locales.
   static const List<Locale> supportedLocales = <Locale>[
-    Locale('en'),
     Locale('de'),
+    Locale('en'),
```
Comment (erdemyerebasmaz) [inline lib/generated/breez_translations.dart:99]: Could you revert this order as it causes:
- #1

More explanation here: https://github.com/breez/Breez-Translations/commit/9c8cc35d9e289a9696ab15ad91734ff12739b171

Comment (ubbabeck) [reply lib/generated/breez_translations.dart:99]: Rebased to main, does it look correct now ?

Comment (erdemyerebasmaz) [reply lib/generated/breez_translations.dart:99]: No, rebasing won't fix this as you're still adding this change on top of main. You need to manually revert the changes with another commit.

File: lib/app_fi.arb [hunk 2]
```diff
@@ -2154,7 +2154,8 @@
     }
   },
   "on_chain_payment_error_not_enough_funds": "Varat eivät riitä.",
-  "valid_payment_error_exceeds_the_limit": "Maksu ylittään rajan {amount}",
+  "valid_payment_error_exceeds_limit":"Maksu ylittää rajan.",
```
Comment (ubbabeck) [inline lib/app_fi.arb:2157]: @tlindi is this translation correct ?

**Review by ademar111190 — APPROVED** [review]
Great job 👏 👏 👏 👏

File: lib/app_fi.arb (hunk 2 above)
Comment (tlindi) [reply lib/app_fi.arb:2157]: Yes

**Pull request merged**